backend/hardware/reagentdispenser/base.py

```diff
@@ -33,3 +33,31 @@ def getPumpSpeedLimits(pumpId):
                     Maximum speed the pump can dispense in ml/s
         """
         pass
+
+    @classmethod
+    def grblWrite(cls, grblSer, command, retries=3):
```

Comment:
The implementation of this seemed to be the same from what I saw in both peristalticpump.py and syringepump.py so I moved it into their parent class.